### DISCUSSION TITLE
Questions about parameter identification process.

### DISCUSSION THREAD

**Discussion by Tarya633** [Open-Systems-Pharmacology/Forum#1477] [Questions & Problems]
1. Why there's no "Parameter Identification" option when the Simulation is based on "Population", though I can make the "Parameter Identification" when the simulation is built with "Individual".
2. Based on "Individual" simulation, I did the "Parameter Identification", but sometimes there was no comparison chart in the field of "Error history", "Predicted vs. Observed" and "Time Profile", I want to know the reason and the handling methods.
Thanks so much.

**Comment by tobiasK2001** [comment]
Hi,

the "standard workflow" in PK-sim is to optimize relevant processes in the simulation for an individual and than extrapolate it to a population. Other ways are currently not possible.
Regarding 2.) not sure why this sometimes happens. Is the PI is running to the end? like the notifacition in the screenshot shows up?
![image](https://user-images.githubusercontent.com/25585248/225031554-e46ff961-8aca-455b-8c00-6033504bcc67.png)
 I can only reproduce it if I uncheck the "refresh option" on the top left on the window.
Or if I use non standard methods like the multiple optimizations with randomized start values
![image](https://user-images.githubusercontent.com/25585248/225032407-dfea5240-cf8c-404b-94f9-be05dae81f22.png)

Hope this might help
Best, Tobias

**Reply by Tarya633** [reply]
Hi Tobias,

Many thanks for your answers. It's a clear for the first question. But the 2rd answer still couldn't solve my confusion, because no matter I checked or unchecked the "refresh" option, there was no pic displayed when PI was done. And also, I checked the "Configuration" module in PI, I found I didn't change any default settings, I mean the "option" in "Configuration" module is "Standard". The screenshot is as follows.

![PI-1](https://user-images.githubusercontent.com/127810809/225188709-b4b5999a-a666-49d0-b311-d9a46f5c44ff.png)

![PI-2](https://user-images.githubusercontent.com/127810809/225195620-ed14f7af-561e-4b2b-8493-08fe167f43ab.png)

I'm very grateful for your help. Thank you so much.

Best, Tarya

**Comment by tobiasK2001** [comment]
Hi Tarya,

looks like your PI finishes very fast after only one second. That is probably to short for generating a history. Did you use Levenberg Marquard as algorithm? Try to change the start value for specific clearance to 1 and use Monte Carlo as algorithm. Lets see what you get than.

Best, Tobias

**Reply by Tarya633** [reply]
Hi Tobias,

Thanks for your advice. I did use the L-M algorithm and selected the "Standard" option. based on your tips, I could get the comparison pictures everytime with Monte Carlo but couldn't with L-M algorithm. It's amazing. 
Also, I checked it in the OSP manual and I didn't find the information that mentions it. Do you know the reason? 
Really appreciated.

Best, Tarya

**Comment by tobiasK2001** [comment]
Hi, I guess you see the pictures with MC as it runs several cycles before getting the final result. The LM is faster and probably finishes already after the first round. So not seeing the history seems to be no error but be design and it is probably not so relevant. Important are the final results. They should be very similar with LM and MC.

Best, Tobias

**Reply by Tarya633** [reply]
Yeah. I agree with you, Tobias. Thank you again for clearing up my confusion. 
Best,Tarya
